README.md

```diff
@@ -2,97 +2,160 @@
+  
+        ```
+        python manage.py generatemodels
+        ```
+    1. Create and execute migration files to initialize your database
+  
+        ```
+        python manage.py makemigrations
+        python manage.py migrate
+        ```
+    1. Create an admin user so that you can log in to the admin dashboard.
+       ```
+       python manage.py createsuperuser --username "username" --email "email@email-address.com"
+       ```
+
+1. In your browser, open the admin dashboard at [http://localhost:8000/admin](http://localhost:8000/admin).
+
+    > **_NOTE:_** Make sure that port 8000 isn't blocked by another service on your host machine.
+
+1. Enter the credentials for the admin user that you created in step 5.3 and log in.
+
+
+
+
+# How to set up your own Herbie project
 
+## 1. Either fork or clone this repository
 
-_Note:_ Please make sure that port 8000 is not blocked by another service on your host-machine.
+We suggest you to fork the project, 
+because you'll be able to make custom modifications (e.g. change the messaging provider) but you'll also get changes from the official repository.
 
-# How to setup herbie for your project?
-### 1 Clone/Fork
-In order to setup a Herbie-Project from scratch you can either fork or clone the repository. We suggest you to fork the project, 
-because like that you have a separate repository so you can make custom modifications (e.g. change the messaging provider) 
-and at the same time receive changes from the official repository in an easy manner
+## 2. Define your business entities and integrate them with Herbie
+    
+You define your business entities with JSON schemas. There are two ways to define your business entities:
 
-### 2 Add Business Entities Schemas Package
-Next step is to define your schemas for your business entities and integrate them with Herbie. There are 2 choices, having a different
-repository for your schemas and load them to herbie as a python package with pip, or adding them directly to the Herbie as a python package
+1. Create a separate repository for your schemas and use pip to load them into Herbie as a Python package.
+
+1. Store your schemas in a Python package that lives directly in your Herbie repository.
+
+
+### Package structure
+
+Regardless of your preferred option, you should structure your package according to the following template:
 
-##### Business Entities Schemas package folders structure
 ```
-.
-└── business_entities_schemas
-    ├── init.py 
-    ├── business_entity1
-    │   ├── business_entity1_v1.json
-    │   └── business_entity1_v2.json    
-    └── business_entity2
+---business_entities_schemas
+        init.py 
+        ------  business_entity1
+                    ------- business_entity1_v1.json
+                    ------- business_entity1_v2.json    
+        ------  business_entity2
```

Comment:
I updated the presentation of the directory structure here. So maybe you can remove this update.

Reply:
@tobi812 OK, added in your changes